dvc/repo/plots/__init__.py

```diff
@@ -357,6 +358,10 @@ def infer_data_sources(plot_id, config=None):
     if isinstance(x, dict):
         sources.append(first(x.keys()))
 
+    # annotation = config.get("annotations", None)
+    # if isinstance(annotation, str):
+    #     sources.append(annotation)
+
```

Comment:
Does this need to get dropped or uncommented?

Reply:
I kept it because Studio needs it to work :)